### DISCUSSION TITLE
I've been trying to install the correct things for playing DRM content but I'm stuck

### DISCUSSION THREAD

**Discussion by pityalvarez** [Eloston/ungoogled-chromium#1930] [Q&A]
I have to install widevine but I'm forced to have google chrome when I don't want to. 
Is it a dll? Is there a way to install widevine without having to do what I mentioned? It's getting difficult for me to understand

**Comment by jstkdng** [comment]
yes, widevine is a dll and some files that are shipped with google chrome, in order to use it you have to copy that dll to the ungoogled chromium install location.
